modules/apps/sync/sync-engine/src/main/java/com/liferay/sync/engine/document/library/handler/BaseSyncDLObjectHandler.java

```diff
@@ -1,85 +0,0 @@
-/**
```

Comment:
Moved this method to GSDLOUH since it's removed from UpdateFileEntry/FolderHandler